EStore/Cart.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+
+namespace EStore
+{
+    public class Cart:IDiscount
+    {
+        List<CartItem> cartItems = new List<CartItem>();
+        double discount=0;
+        public void AddItem(CartItem cartItem)
+        {
+            //CartItem cartItem = new CartItem(product, quantity);
+            CartItem _cartItem= cartItems.Find(item => item.product == cartItem.product);
```

Comment:
To check if an element already exist in a cart, create separate function for it so that it can be reused elsewhere also.
The function should return a boolean value for condition check.

Reply:
Resolved